lib/messages.js

```diff
@@ -39,16 +39,27 @@ const RESTORE_STASH_EXAMPLE = `  Any lost modifications can be restored from a g
     > git stash apply --index stash@{0}
 `
 
+const UNSAFE_SHELL = chalk.yellow(`${warning} Using the shell option is unsafe and might lead to command injection!
```

Comment:
We haven't used such a prefix in any ouput, so I wouldn't add just for this case. Maybe this can be revisited later, but would require a comprehensive look of the whole logging logic.

Reply:
The problem I'm kind of expecting to occur, is that the warning will frustrate people who just updated `lint-staged` as part of a bigger build process, and who suddenly have a new warning of which the origin is unclear. I'd argue that it'd be useful to add a prefix here because of that, even if that's not done elsewhere, just to reduce the frustration of unexpected warnings on the user's side.